prime-router/src/testIntegration/resources/datatests/mappinginventory/catchall/cx/cx-identifier-system-iso-no-cx-4-2.fhir

```diff
@@ -125,7 +128,14 @@
     "fullUrl" : "ServiceRequest/1710884721570486000.84bb48da-4f4a-456c-96ab-dd772cfd7283",
     "resource" : {
       "resourceType" : "ServiceRequest",
-      "id" : "1710884721570486000.84bb48da-4f4a-456c-96ab-dd772cfd7283",
+      "id" : "1721668788639392000.7b4a51e9-f5b3-455b-ab0f-443a3d11076f",
```

Comment:
It looks like this is happening in a few files. I think it might be the cause of the test failures I'm seeing. This id was changed, but it wasn't updated in the "fullUrl" on the line above, and the reference to it elsewhere in the file wasn't updated.

Reply:
It was not the cause of the test failures (that was all me, leftover stuff in the db from another branch), but does still seem not great...

Reply:
Thanks for pointing this out! These kinds of oddities/incorrectness in test data are a pet peeve so I was horrified to learn that the call was coming from inside the house: https://github.com/CDCgov/prime-reportstream/pull/15333/commits/ee572cb92c26dce7f42161780f590b9d32073711